src/redux/store.js

```diff
@@ -24,11 +25,9 @@ const middlewares = [
   HttpMiddleware,
   PushNotificationMiddleware,
   CentrifugoMiddleware,
-  GeolocationMiddleware,
-  BluetoothMiddleware,
-  ringOnNewOrderCreated,
-  ringOnTaskListUpdated,
```

Comment:
If `react-native-background-geolocation` is disabled, can we keep the `GeolocationMiddleware`, or it will break?

Reply:
Maybe, we should include middlewares based on if it is a custom build or not?